codespell_lib/data/dictionary.txt

```diff
@@ -20874,12 +20892,15 @@ programable->programmable
 programatic->programmatic
 programatically->programmatically
 programattically->programmatically
+programd->programmed
+programed->programmed
 programemer->programmer
 programemers->programmers
 programers->programmers
 programing->programming
-programm->program
+programm->program, programme,
 programmaticaly->programmatically
+programmd->programmed
```

Comment:
I think this should suggest programme too given the d/e proximity.

Reply:
added and rebased.